BitwardenShared/Core/Auth/Models/Response/MasterPasswordPolicyResponseModel.swift

```diff
@@ -1,6 +1,19 @@
 /// API response model for master password policies.
 ///
 struct MasterPasswordPolicyResponseModel: Codable, Equatable {
+    // MARK: Types
+
+    /// Key names used for encoding and decoding.
+    enum CodingKeys: String, CodingKey {
+        case enforceOnLogin = "EnforceOnLogin"
+        case minComplexity = "MinComplexity"
+        case minLength = "MinLength"
+        case requireLower = "RequireLower"
+        case requireNumbers = "RequireNumbers"
+        case requireSpecial = "RequireSpecial"
+        case requireUpper = "RequireUpper"
+    }
```

Comment:
@andrebispo5 I believe this is because `IdentityTokenErrorModel` doesn't conform to `JSONResponse`, thus it doesn't use the `pascalOrSnakeCaseDecoder`.
I think we could just change: 
`try? JSONDecoder().decode` 
for 
`try? JSONDecoder.pascalOrSnakeCaseDecoder.decode`
in here and that should fix the issue without the `CodingKeys`
https://github.com/bitwarden/ios/blob/1f97eabc83c4c8d61109036bf34af574d32f3c4e/BitwardenShared/Core/Auth/Services/API/Auth/Requests/IdentityTokenRequest.swift#L80-L84

Reply:
do you agree @matt-livefront ?